demo.yaml/beam.yaml

```diff
@@ -0,0 +1,9 @@
+%YAML 1.2
+---
+class: BeamMockup.BeamMockup
+configuration:
+  defaultBeamDivergence: (12.0, 5.0)
```

Comment:
I think this should rather be
```
   beam_divergence_vertical: 12.0
   beam_divergence_horizontal: 5.0
```
Ad eventually:
  `     definer_type: aperture`

Reply:
Yes, that was it, thanks!